### ISSUE TITLE
Explain integration of Avalonia with mobile (xamarin) and web (.NET WASM) toolings

### ISSUE THREAD

**Issue by maxkatz6** [AvaloniaUI/Documentation#423]
It's easy to misunderstand what is Avalonia's role on mobile and look for various packaging related documentation pages.
When in fact Avalonia does not do any packaging, so all the signing, icon related, dependencies questions should be redirected to the official Microsoft tooling. 

For example, "How to add icon to Avalonia iOS app" should redirect users to the https://learn.microsoft.com/en-us/xamarin/ios/app-fundamentals/images-icons/app-icons?tabs=macos page ("How to add icon to .NET iOS app").

**Comment by maxkatz6** [comment]
It is a bit confusing, as old Xamarin.Native and Xamarin.Forms are now deprecated.

But in fact, new .NET mobile runtime (starting with .NET 6) is in fact the same Xamarin.Native with modernizations, meaning old documentation is still applicable.
And instead Xamarin.Forms there is now MAUI.

Meaning, both MAUI and Avalonia work on top of .NET mobile runtime (formerly Xamarin.Native)

**Comment by davidchavezgrant** [comment]
Agreed this would be extremely helpful as a new user, especially coming from Xamarin / MAUI which do handle all of these things.

**Comment by maxkatz6** [comment]
Also, to clarify more confusion, MAUI specifically implemented an abstraction for some features like app icons. https://learn.microsoft.com/en-us/dotnet/maui/user-interface/images/app-icons?view=net-maui-7.0&tabs=android .
This **won't** work in Avalonia, as it is MAUI specific addition not available in plain Xamarin.Native (nor .net 6 mobile).
Unless MS makes these API available everywhere which is less likely.
